crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -775,6 +779,13 @@ pub struct ParseState {
     // ASSERT_MY_BIRTH_HEIGHT
     // each item is the index into the SpendBundleConditions::spends vector
     assert_not_ephemeral: HashSet<usize>,
+
+    // All public keys and messages emitted by the generator. We'll validate
+    // these against the aggregate signature at the end, unless the
+    // DONT_VALIDATE_SIGNATURE flag is set
+    // TODO: We would probably save heap allocations by turning this into a
+    // blst_pairing object.
+    pub pkm_pairs: Vec<(PublicKey, Vec<u8>)>,
```

Comment:
Maybe using the `Bytes` type here would be ideal for consistency and to make `Debug` printing cleaner?